### PR TITLE
update models.py

### PR DESCRIPTION
**Pull request by krishnanjali25**

### CODE REVIEW

**Comment by arshadsid** [comment]
Still not compliant

**Comment by krishnanjali25** [comment]
when i have run the command isort i kept all y for all the changes .

**Comment by arshadsid** [comment]
Click on details you can see...

**Comment by krishnanjali25** [comment]
It is stopping at academic procedures

**Comment by arshadsid** [comment]
The chages yoy made in acad procedure is causing that

File: FusionIIIT/applications/academic_procedures/forms.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
-from django import forms
-
 from applications.academic_information.models import Course
+from django import forms
```
Comment (arshadsid) [inline FusionIIIT/applications/academic_procedures/forms.py:2]: This file is different in the PR and your master I just forked. I dont think you made the PR to the updated version.

**Review by arshadsid — CHANGES_REQUESTED** [review]
Close your PR and make a pull request with all the files that you have changed

**Comment by arshadsid** [comment]
f

**Comment by arshadsid** [comment]
this is from your file while your PR reflects something else.

**Comment by krishnanjali25** [comment]
It is there in main repository also
screenshot from 2017-10-16 17-39-01
screenshot from 2017-10-16 17-40-36

**Pull request closed without merging**